docs/play/getting-started_cpp.md

```diff
@@ -23,10 +23,10 @@ Please download the archive file based on your OS in the [release page](https://
 
 #### Windows
 
-1. Clone the current repository
-   ```sh
-   git clone https://github.com/cronos-labs/play-cpp-sdk.git
-   ```
```

Comment:
May I know why remove the indent?

Original:
<img width="571" alt="image" src="https://user-images.githubusercontent.com/92778185/175273291-fe74a93d-21d9-4c44-a1a6-52d4e2d510b4.png">
Now:
<img width="550" alt="image" src="https://user-images.githubusercontent.com/92778185/175273481-39bd9257-fcca-4a84-8a8a-e97d7cb70196.png">

Reply:
We can put the indent back as before, but then I have to change it on the other pages as well to make it consistence. What do you think?

Reply:
@damoncro fixed